automatic/ffmpeg-full/README.md

```diff
@@ -10,4 +10,5 @@ Security is a high priority and code review is always done with security in mind
 ## Notes
 
 * This package is for the stable static full ffmpeg version.
-
+* For a list of libraries included, go to https://www.gyan.dev/ffmpeg/builds/#libraries
```

Comment:
Just out of curiosity, should we have a mention here that the libraries listed on that page may only be valid for the latest version of the package?

Thoughts @TheCakeIsNaOH @gep13

Reply:
Yup, agreed that it should note that the list of libraries is for the current latest release.

Reply:
I added a note for the fact that the list may only apply to the lastest version

It's now in the same branch as I used for this PR.